.github/workflows/release.yaml

```diff
@@ -1,112 +0,0 @@
-name: Release
-on:
-  workflow_dispatch:
-    inputs:
-      version:
-        type: string
-        description: The version you intend to release (eg x.y.z)
-        required: true
-  pull_request:
-    types: [ closed ]
-  push:
-    branches:
-      - 'release/**'
-    tags:
-      - v*
-
-env:
-  VERSION: ${{ github.event.inputs.version }}
-  APP_ID: 257262
```

Comment:
No longer need to exchange app token because both both workflows (create pr and draft release) are triggered by human action.